tests/test_emails.py

```diff
@@ -233,3 +234,45 @@ def test_send_email(self):
                 recipients=['lab@email.com', 'facility@email.com', 'finance@email.com', 'project@email.com'],
                 strict=True
             )
+
+class TestDataReleaseFacilityManager(TestEmailEPP):
+    def setUp(self):
+        super().setUp()
+        self.epp = self.create_epp(DataReleaseFMEmail)
+
+    def test_only_one_project(self):
+        try:
+            with self.assertRaises(ValueError):
+                with self.patch_project_multi:
+                    self.epp._run()
+
+        except NotImplementedError:
+            print('Skipping test for abstract class: ' + self.epp.__class__.__name__)
```

Comment:
This is exactly the same method as in the superclass, so can be removed. `TestDataReleaseFacilityManager` will inherit the method from the superclass and run the test